justfile

```diff
@@ -13,21 +13,29 @@ test:
     cargo clippy --workspace
     cargo test --workspace
 
-run-wascc: _cleanup_kube
+run-wascc: _cleanup_kube bootstrap-ssl
     # Change directories so we have access to the ./lib dir
-    cd ./crates/wascc-provider && cargo run --bin krustlet-wascc --manifest-path ../../Cargo.toml
+    cd ./crates/wascc-provider && cargo run --bin krustlet-wascc --manifest-path ../../Cargo.toml -- --pfx-path "../../config/certificate.pfx"
 
-run-wasi: _cleanup_kube
+run-wasi: _cleanup_kube bootstrap-ssl
     # HACK: Temporary step to change to a directory so it has access to a hard
     # coded module. This should be removed once we have image support
-    cd ./crates/wasi-provider && cargo run --bin krustlet-wasi --manifest-path ../../Cargo.toml
+    cd ./crates/wasi-provider && cargo run --bin krustlet-wasi --manifest-path ../../Cargo.toml -- --pfx-path "../../config/certificate.pfx"
 
 dockerize:
     docker build -t technosophos/krustlet:latest .
 
 push:
     docker push technosophos/krustlet:latest
 
+bootstrap-ssl:
+    mkdir -p config
+    rm -rf config/*
```

Comment:
This should probably check if the cert already exists before doing the steps every time. Not a blocker though